Entitas/Entitas/CodeGenerator/Generators/ComponentExtensionsGenerator.cs

```diff
@@ -22,6 +22,10 @@ public class ComponentExtensionsGenerator : IComponentCodeGenerator {
 
         static string generateComponentExtension(ComponentInfo componentInfo) {
             var code = addNamespace();
+            if (componentInfo.isSingleEntity == false) {
+                code += addEntityInterface(componentInfo);
+            }
+
```

Comment:
Atm, I'm not generating interfaces for single entities. I'll remove this constraint soon.